addons/hr_expense/i18n/hr_expense.pot

```diff
@@ -1767,13 +1788,6 @@ msgid ""
 "expenses for you."
 msgstr ""
 
-#. module: hr_expense
-#. odoo-python
-#: code:addons/hr_expense/models/hr_expense.py:0
-#, python-format
-msgid "Specify expense journal to generate accounting entries."
-msgstr ""
-
```

Comment:
You shouldn't remove translation terms in stable. No worries if it's now unused; just let it there.